Frends.SFTP.WriteFile/README.md

```diff
@@ -0,0 +1,68 @@
+# Frends.SFTP.WriteFile
+
+[![Frends.SFTP.WriteFile Main](https://github.com/FrendsPlatform/Frends.SFTP/actions/workflows/WriteFile_build_and_test_on_main.yml/badge.svg)](https://github.com/FrendsPlatform/Frends.SFTP/actions/workflows/WriteFile_build_and_test_on_main.yml)
+![MyGet](https://img.shields.io/myget/frends-tasks/v/Frends.SFTP.WriteFile?label=NuGet)
+![GitHub](https://img.shields.io/github/license/FrendsPlatform/Frends.SFTP?label=License)
+![Coverage](https://app-github-custom-badges.azurewebsites.net/Badge?key=FrendsPlatform/Frends.SFTP/Frends.SFTP.WriteFile|main)
+
+Writes a file to SFTP server.
+
+## Installing
+
+You can install the task via FRENDS UI Task View or you can find the NuGet package from the following NuGet feed
+
+### Source
```

Comment:
Source, Destination, Connection and Result documentation can be removed, since they are automatically generated to Frends Task Portal.

Reply:
Fixed